### PR TITLE
feat: [PIE-3626]: cypress parallel runs

### PR DESCRIPTION
**Pull request by vkbansal**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vkbansal** [comment]
retrigger titlecheck

**Comment by vkbansal** [comment]
retrigger titlecheck

**Review by chetanvekariya-harness — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -241,7 +226,7 @@
     "webpack-retry-chunk-load-plugin": "^3.0.0",
     "worker-loader": "^3.0.8",
     "yaml-loader": "^0.6.0",
-    "zx": "^4.3.0"
+    "zx": "^5.3.0"
```
Comment (abhinavrastogi-harness) [inline package.json:229]: do we need any sec approval or plan for this upgrade?

**Review by abhinavrastogi-harness — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (vkbansal) [reply package.json:229]: This used internally and not shipped to end user.

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**